app/assets/javascripts/discourse/app/components/glimmer-header/topic/info.gjs

```diff
@@ -0,0 +1,187 @@
+    return _additionalFancyTitleClasses.join(" ");
+  }
+
+  get twoRows() {
+    return (
+      this.tags?.length ||
+      this.showPM ||
+      this.siteSettings.topic_featured_link_enabled
+    );
+  }
+
+  get tags() {
+    if (this.args.topic.tags) {
+      return renderTags(this.args.topic);
+    }
+  }
+
+  get remainingParticipantCount() {
+    return this.totalParticipants - this.maxExtraItems;
+  }
+
+  get participants() {
+    const participants = [
+      ...this.args.topic.details.allowed_users,
+      ...this.args.topic.details.allowed_groups,
+    ];
+    return participants.slice(0, this.maxExtraItems);
+  }
+
+  @action
+  jumpToTopPost(e) {
+    e.preventDefault();
+    if (this.args.topic) {
+      DiscourseURL.routeTo(this.args.topic.firstPostUrl, {
+        keepFilter: true,
+      });
+    }
+  }
+
+  <template>
+    <div
+      class={{concatClass (if this.twoRows "two-rows") "extra-info-wrapper"}}
+    >
+      <div class={{concatClass (if this.twoRows "two-rows") "extra-info"}}>
+        <div class="title-wrapper">
+          <h1 class="header-title">
+            {{#if this.showPM}}
+              <a
+                class="private-message-glyph-wrapper"
+                href={{fn this.currentUser.pmPath @topic}}
+                aria-label={{i18n "user.messages.inbox"}}
+              >
+                {{icon "envelope" class="private-message-glyph"}}
+              </a>
+            {{/if}}
+
+            {{#if (and @topic.fancyTitle @topic.url)}}
+              <Status @topic={{@topic}} @disableActions={{@disableActions}} />
+
+              <a
+                class={{concatClass
+                  "topic-link"
+                  this.additionalFancyTitleClasses
+                }}
+                {{on "click" this.jumpToTopPost}}
+                href={{@topic.url}}
+                data-topic-id={{@topic.id}}
+              >
+                <span>{{htmlSafe @topic.fancyTitle}}</span>
+              </a>
+
+              <span class="header-topic-title-suffix">
+                <PluginOutlet
+                  @name="header-topic-title-suffix"
+                  @outletArgs={{hash topic=@topic}}
+                />
+              </span>
+            {{/if}}
+          </h1>
+
+          {{#if (or @topic.details.loaded @topic.category)}}
+            {{#if
+              (and
+                @topic.category
+                (or
+                  (not @topic.category.isUncategorizedCategory)
+                  (not this.siteSettings.suppress_uncategorized_badge)
+                )
+              )
+            }}
+              <div class="categories-wrapper">
+                {{#if @topic.category.parentCategory}}
+                  {{#if
+                    (and
+                      (@topic.parentCategory.parentCategory)
+                      (not this.site.mobileView)
+                    )
+                  }}
+                    {{categoryLink @topic.category.parentCategory}}
```

Comment:
I think this needs to be `category.parentCategory.parentCategory` ? Otherwise we're showing `parentCategory` twice.